docs/spec_draft.md

```diff
@@ -198,8 +198,8 @@ described below)
 ### Semantics
 
 Performs element-wise absolute value of `operand` tensor and produces a `result`
-tensor. For floating-point element types, it implements the `abs` operation from
-the IEEE-754 specification.
+tensor. For floating-point type, it implements the `abs` operation from the
```

Comment:
I think that the plural form (i.e. "types") reads better than the singular form (i.e. "type") outside of the tables.

In my mind, "tensor of signed integer, floating-point, or complex type" reads better with singular because for every given tensor there's just one element type.

To the contrary, I think that "For floating-point types, it implements the `abs` operation from the IEEE-754 specification" reads better with plural because it applies to multiple floating-point types at once (`bf16`, `f16`, etc). When I read "For floating-point type, it implements the `abs` operation", what I find asking myself is "for what type specifically?".

Reply:
Thanks for the explanation. Got it!